packages/mwp-app-render/src/components/SEOHead.jsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Helmet from 'react-helmet';
+
+import { DEFAULT_TITLE } from 'mwp-app-render/src/util/seo/ldJson';
+import { generateCanonicalUrlMetaTags } from 'mwp-app-render/src/util/seo/links';
+import {
+	DEFAULT_IMAGE_URL,
+	generateMetaData,
+	generateMetaTags,
+} from 'mwp-app-render/src/util/seo/metaTags';
+
+/**
+ * Component for rendering SEO content in the document's head
+ * Note: props passed to SEOHead should *not* be html-escaped because this is handled by Helmet
+ * @see https://github.com/nfl/react-helmet
+ * @param {String} baseUrl Base url of the page (protocol + hostname).
+ * @param {String} imageUrl An image url that will be applied to the link tag, og:image meta tag, etc.
+ * @param {Array} ldJson Array containing json-ld objects. These are usually generated by helper functions in src/util/seoHelper.js.
+ * @param {String} localCode The current localeCode.
+ * @param {String} ogDescription A special description for the Open Graph og:description meta tag. If this isn't provided pageDescription is used.
+ * @param {String} ogTitle A special title for the Open Graph og:title meta tag. If this isn't provided pageTitle is used.
+ * @param {String} pageDescription The page description, for example -- Upcoming Events for NY Tech in New York City
+ * @param {String} pageKeywords A comma-separated string of keywords, for example -- 'upcoming events,ny-tech,ny,technology,group,club,event,community,local,networking,meet,sharing'
+ * @param {Array} pageMeta Optional meta tag data, for example -- [{name: 'yandex', value: 12345},{name: 'bing', value: 12345}]
+ * @param {String} pageTitle Content for the page's title tag
+ * @param {Boolean} robots Instructs search engines whether or not we'd like them to crawl the pages and its links
+ * @param {String} route The current route
+ * @module SEOHead
+ */
+export const SEOHeadComponent = ({
+	baseUrl,
+	imageUrl,
+	ldJson,
+	localeCode,
+	ogDescription,
+	ogTitle,
+	pageDescription,
+	pageKeywords,
+	pageMeta,
+	pageTitle,
+	robots,
+	route,
+}) => {
+
+	const metaData = generateMetaData({
+		appPath: `meetup:/${route}`,
+		baseUrl,
+		description: pageDescription,
+		imageUrl,
+		localeCode,
+		keywords: pageKeywords,
+		ogTitle,
+		ogDescription,
+		route,
+		title: pageTitle,
+	});
+
+	const metaTags = generateMetaTags([...metaData, ...pageMeta]);
+
+	const canonicalUrlMetaTags = generateCanonicalUrlMetaTags(baseUrl, localeCode, route);
+
+	const ldJsonTags = ldJson.map((jsonObj, index) => (
+		// eslint-disable-next-line react/no-array-index-key
+		<script type="application/ld+json" key={`ldjson-${index}`}>
+			{JSON.stringify(jsonObj)}
+		</script>
+	));
+
+	return (
+		<Helmet defaultTitle="Meetup" titleTemplate="%s | Meetup">
```

Comment:
might be nice to make this flexible so that it could be either `Meetup` or `Meetup Pro`